src/webgpu/api/validation/queue/copyToTexture/CopyExternalImageToTexture.spec.ts

```diff
@@ -1,36 +1,14 @@
 export const description = `
-copyImageBitmapToTexture Validation Tests in Queue.
-TODO: Should this be the same file as, or next to, web_platform/copyImageBitmapToTexture.spec.ts?
-
-TODO: Split this test plan per-test.
-
-Test Plan:
-- For source.imageBitmap:
-  - imageBitmap generated from ImageData:
-    - Check that an error is generated when imageBitmap is closed.
-
-- For destination.texture:
-  - For 2d destination textures:
-    - Check that an error is generated when texture is in destroyed state.
-    - Check that an error is generated when texture is an error texture.
-    - Check that an error is generated when texture is created without usage COPY_DST.
-    - Check that an error is generated when sample count is not 1.
-    - Check that an error is generated when mipLevel is too large.
-    - Check that an error is generated when texture format is not valid.
-
-- For copySize:
-  - No-op copy shouldn't throw any exception or return any validation error.
-  - Check that an error is generated when destination.texture.origin + copySize is too large.
-
-TODO: copying into slices of 2d array textures. 1d and 3d as well if they're not invalid.
+copyExternalImageToTexture Validation Tests in Queue.
```

Comment:
I remembered that for canvas elements we also need to make sure they are not 'placeholder', i.e. they are not a canvas that has had transferControlToOffscreen called on it. Please add a TODO for this - and you can follow up with an implementation in another PR if you'd like.